docroot/themes/humsci/su_humsci_theme/patterns/molecules/hero-text-overlay/scss/hero-text-overlay.scss

```diff
@@ -0,0 +1,252 @@
+.hero-text-overlay {
+  position: relative;
+  .image {
+    &:before {
+      bottom: 0;
+      content: "";
+      display: flex;
+      height: 100%;
+      position: absolute;
+      width: 100%;
+      z-index: 1;
+    }
+    &.color-default:before {
```

Comment:
Combine this with the rule above. so if there is no class it will always have a color